metrics/metrics.go

```diff
@@ -41,56 +41,55 @@ var (
 	SentBytes              = stats.Int64("libp2p.io/dht/kad/sent_bytes", "Total sent bytes per RPC", stats.UnitBytes)
 )
 
-// Views
-var (
-	ReceivedMessagesView = &view.View{
+var Views = []*view.View{
```

Comment:
This changes means consumers of this package will no longer be able to register just _some_ of the views, is that correct? 

I liked having the ability to pick and choose which views my code registered, for example maybe I am only interested in `ReceivedBytesView` and `SentBytesView`. We could preserve this functionally by leaving the views assigned to exported variables while also adding them to an exported slice, as was done here. What do you think?

Reply:
That seems reasonable, but I think it binds the API to particular views that may not have relevance going forward. I'm not sure there's ever a circumstance where all the views would not be exported (now I think perhaps `DefaultViews` would be a better name). They can still be found in the slice, and are self-describing. Further, should users want to roll their own views, or modify them, they can do that, the measures are exported for doing so. I suspect in future, that with all the libp2p and other packages having various metrics and views exposed, we'll need to switch to some automatic registration anyway to avoid fatigue traversing all the packages of interest and selecting individual views.

If you feel strongly about it, I'll leave them exported for now, and provide the slice just for convenience.

Reply:
No strong feelings, you make a good point about binding the API to views that may become irrelevant -- this lgtm as is.